src/siema.js

```diff
@@ -305,6 +319,7 @@ export default class Siema {
     this.selector.removeEventListener('mouseup', this.mouseupHandler);
     this.selector.removeEventListener('mouseleave', this.mouseleaveHandler);
     this.selector.removeEventListener('mousemove', this.mousemoveHandler);
+    document.removeEventListener('keydown', this.keydownHandler);
```

Comment:
Sorry @znk but hijacking `keydown` event on document body may be dangerous. It can harm a lot core functionality on the project and it's tricky from accessibility point of view as well. I prefer users who are conscious what they do, to do it by themselves (if they really have to). Personally I consider it as an anti-pattern on majority of cases. The only reasonable use case is having a full page carousel and nothing else on the page (like a presentation mode).